internal/access/access_key.go

```diff
@@ -89,12 +87,15 @@ func DeleteAccessKey(c *gin.Context, id uid.ID, name string) error {
 	if key.IssuedFor == rCtx.Authenticated.User.ID {
 		// users can delete their own keys
 	} else {
-		_, err := RequireInfraRole(c, models.InfraAdminRole)
-		if err != nil {
+		if err := IsAuthorized(rCtx, models.InfraAdminRole); err != nil {
```

Comment:
`RequireInfraRole` calls `IsAuthorized`, so this is not a change in behaviour. The only difference between these functions is one accepts `gin.Context` and returns a DB, while the other accepts `RequestContext` and expects the caller to use the db from the `RequestContext`.

 I had to change from one to the other because I removed `gin.Context`.